algorithmic_efficiency/workloads/imagenet_resnet/imagenet_pytorch/workload.py

```diff
@@ -154,7 +155,6 @@ def init_model_fn(
     model.to(DEVICE)
     if N_GPUS > 1:
       if USE_PYTORCH_DDP:
-        model = torch.nn.SyncBatchNorm.convert_sync_batchnorm(model)
```

Comment:
I'm a pytorch noob so I may be misunderstanding, but from my reading of the email thread, now that we delete the sync BN from here does that mean we _have_ to use DDP in order to get BN stats to sync at all? should we throw an error if `USE_PYTORCH_DDP == False` in the imagenet workloads?

Reply:
In my understanding, `DataParallel ` updates the BN stats on the default device and we don't need to use synchronization (https://discuss.pytorch.org/t/understanding-behavior-of-torch-nn-dataparallel/91676).